index.html

```diff
@@ -0,0 +1,128 @@
+            <p>@Gwangalli Beach, Busan</p>
+        </div>
+    </section>
+    <section id="main-program">
+        <h3 class="including">Including</h3>
+        <div class="include-container">
+        <ul class="include-list">
+            <li><i class="fa-sharp fa-solid fa-music"></i></li>
+            <li class="offer">Music</li>
+            <li>We made history by discovering new musical promises and presenting the most renowned names in music worldwide. With a careful and coherent lineup, come join us in this adventure and share the message of music.</li>
+        </ul>
+        <ul class="include-list">
+            <li><i class="fa-solid fa-campground"></i></li>
+            <li class="offer">Camping</li>
+            <li>Camping is the ultimate festival experience. You can camp for free or if you want a bit more comfort book your spot in our upgrade campsites with accommodations!</li>
+        </ul>
+        <ul class="include-list">
+            <li><i class="fa-solid fa-utensils"></i></li>
+            <li class="offer">Food & drinks</li>
+            <li>We will have over 400 food traders offering an array of exciting eats for Festival-goers. The choices are endless with a variety of delicious, world cuisine available across the site.</li>
+        </ul>
+        <ul class="include-list">
+            <li><i class="fa-solid fa-couch"></i></li>
+            <li class="offer">Chill zone</li>
+            <li>The area is meant as a quality chill-out zone. There will be cool daytime concerts and dj-sets during the afternoon and night, along with a bar and smoothie corner.</li>
+        </ul>
+        <ul class="include-list">
+            <li><i class="fa-solid fa-arrow-right-arrow-left"></i></li>
+            <li class="offer">Traders</li>
+            <li>Thousands of food traders, charities, campaigning organisations and sellers of the bizarre and unique apply to trade at our site each year</li>
+        </ul>
+        </div>
+        <div class="purchase-ticket">
+            <button type="button" class="purchase-button">Purchase ticket</button>
+        </div>
+        <div class="whole-program">
+            <a href="" class="program-link">See the whole program</a>
+        </div>
+    </section>
+    <section id="featured-bands">
+
+    </section>
+    <section id="sponsors">
+        <h3 class="sponsors-title">Partners and Sponsors</h3>
+        <div class="sponsors-list">
+            <h6>Coca-Cola</h6>
+            <h6>Red Bull</h6>
+            <h6>Jack Daniels</h6>
+            <h6>ticketswap</h6>
+            <h6>Bolt</h6>
+        </div>
+    </section>
+    <footer id="footer-desktop">
+        <img class="footer-logo" src="images/logo_square_large_print.webp" alt=""></img>
+        <div class="footer-rights">
+            <p>Musicians Without Borders.</p>
+            <p> Some rigths reserved.</p>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit.</p>
+        </div>
+    </div>
```

Comment:
- [ ] As mentioned in the requirements that the pages look almost identical to the original design, kindly consider fixing the following:
 The `Partners and Sponsors` heading text should be similar to the original design and should have a small red divider beneath it.